reactioncommerce/reaction
# [Marketplace] Fix group error on Account Management Screen on Merchant shops #2726;#2728

Resolves #2726 and #2728 

**How To Test:**
* Create a merchant shop
* Visit a merchant shop's account section as Marketplace owner 
* Confirm that you can see the account's section (re: #2726)
* Also confirm that permissions list are not duplicated (re: #2728)

Code Changes Introduced:
* The AdminInvite Form needs at least one "invitable" group to work properly. That was failing when the owner tried to see the merchant dashboard. I've moved that check into the parent component. So there is check that there's an "invitable"group before rendering the form.
* I also added the missing shopId value to fix #2728
